### ISSUE TITLE
[6.2.0-beta2] FrameLayout to show Fragment not found

### ISSUE THREAD

**Issue by tbalcom** [MvvmCross/MvvmCross#3059]
## 🔙 Regression

An application which previously worked in MvvmCross 6.1.2 can no longer navigate to the first ViewModel after upgrading from MvvmCross 6.1.2 to 6.2.0-beta2.

I used MvxScaffolding Visual Studio plugin to generate the project. I changed the `MainContainerActivity` from a basic `FrameLayout` container to a `DrawerLayout` container using `NavigationView` and a `FrameLayout`. The project runs as expected while using MvvmCross 6.1.2. At this point I updated to MvvmCross 6.2.0-beta2 and the project no longer displays the first ViewModel.

The error is: 
> System.NullReferenceException: FrameLayout to show Fragment not found

### Old (and correct) behavior
The first ViewModel is navigated to.

### Current behavior
A `System.NullReferenceException` is thrown during setup. The splash screen displays for a while, then the screen turns black instead of showing the first ViewModel.

### Reproduction steps
I've reproduce the problem [here](https://github.com/tbalcom/FrameLayoutNotFound). The project is currently referencing MvvmCross 6.2.0-beta2 from NuGet so you can observe the reported problem. You can edit the two CSPROJ files to downgrade back to 6.1.2, Clean, Rebuild and notice how the first ViewModel is displayed.
### Configuration

**Version:** 6.2.0-beta2

**Platform:** 
- [ ] :iphone: iOS
- [x] :robot: Android
- [ ] :checkered_flag: WPF
- [ ] :earth_americas: UWP
- [ ] :apple: MacOS
- [ ] :tv: tvOS
- [ ] :monkey: Xamarin.Forms

**Comment by nickrandolph** [comment]
Hey @tbalcom thanks for submitting this bug. If you get a chance would you be able to make changes to the Playground sample and submit as a PR demonstrating the issue - this makes it easier for anyone to jump in and fix the issue

**Comment by tbalcom** [comment]
@nickrandolph I'm not able to reproduce this in the current Playground.Droid project. I can reproduce in projects bringing in MvvmCross 6.2.0-beta2 via NuGet. I've used a project generated with MvxScaffolding and made a new manually created project and it all falls apart when I upgrade from 6.1.2 to 6.2.0-beta2. I guess this means it's possible the problem has since been fixed in the current develop branch? The 6.2.0-beta2 is from almost a month ago so it's different compared to what's currently in the develop branch.
